fvm/programs/invalidator.go

```diff
@@ -18,13 +19,22 @@ type ContractUpdate struct {
 	Code []byte
 }
 
+type MeterParamOverrides struct {
+	ComputationWeights meter.ExecutionEffortWeights // nil indicates no override
+	MemoryWeights      meter.ExecutionMemoryWeights // nil indicates no override
+	MemoryLimit        *uint64                      // nil indicates no override
+}
+
 type ProgramInvalidator DerivedDataInvalidator[
 	common.AddressLocation,
 	*interpreter.Program,
 ]
 
+type MeterParamOverridesInvalidator DerivedDataInvalidator[int, MeterParamOverrides]
```

Comment:
Is this the second table you are adding?

Is the `int` the type of `meterParamOverridesKey`?

Reply:
yes, this is the second table.  as alexey pointed out, i should use `struct{}` instead of `int` as the key since the table is single valued.